opencog/atoms/distvalue/README.md

```diff
@@ -0,0 +1,44 @@
+Distributional Values
+=====================
+
+An implementation of Distributional Values that might eventually replace all current TruthValues.
+
+Issues and Goals
+----------------
+
+One of the problems this is trying to solve is the combination of fuzzy and propabalistc logic. As both are used within OpenCog it is important to not only be able to clearly represent and distinguise them but also use them together in a combined for. That way no matter if the alogrithm expects fuzzy or propabalistc logic values they can work on a DistributionalValue. Which improves interoperability.
+Another issue this improves upon is the accuracy and detail of TruthValues albeit at the cost of higher complexity.
```

Comment:
please wrap lines so that they fit in an 80-column terminal (i.e. wrap at the 72nd column)
Also, please run the spell-checker.